src/main/java/adris/altoclef/tasksystem/ITaskLimitsSlots.java

```diff
@@ -0,0 +1,28 @@
+package adris.altoclef.tasksystem;
```

Comment:
New Interface which when implemented, allows for the clearing of certain slots. Used in ResourceTask to clear the 2x2 grid when not crafting with it.